{"name": "test_registration_with_invalid_data", "status": "passed", "start": 1722047976327, "stop": 1722047977318, "uuid": "6056a12a-d4b5-42ed-abcf-5e76556dfee0", "historyId": "20abe179da2ce0e7e7aabd1e3acbd83e", "testCaseId": "20abe179da2ce0e7e7aabd1e3acbd83e", "fullName": "test.test_Registration.TestLogin#test_registration_with_invalid_data", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup_and_teardown', 'log_on_failure')"}, {"name": "parentSuite", "value": "test"}, {"name": "suite", "value": "test_Registration"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "Raunak"}, {"name": "thread", "value": "10708-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.test_Registration"}]}